test/test_quantization.py

```diff
@@ -420,6 +420,39 @@ def checkQuantized(model):
         model = quantize_dynamic(NestedModel().eval(), test_only_eval_fn, self.calib_data, qconfig_dict)
         checkQuantized(model)
 
+    def test_type_match_rule(self):
+        r"""Test quantization for nested model, top level 'fc3' and
+        'fc1' of submodule 'sub2', All 'torch.nn.Linear' modules are quantized
+        """
+        model = NestedModel().eval()
+        qconfig_dict = {
+            'fc3': default_qconfig,
```

Comment:
Is it possible to have an example where:
qconfig_dict = {
            'fc3': None, 
            torch.nn.Linear:default_qconfig
}
and show that all layers except fc3 are dynamically quantized?

Reply:
Updated. In the new version, we choose not to quantize `fc3` and `sub2.fc1`. We show `sub1.fc` and `sub2.fc2` are dynamic quantized, while `fc3` and `sub2.fc1` remain the same.